lua/default-config.lua

```diff
@@ -14,23 +14,23 @@ lvim = {
   database = { save_location = "~/.config/lunarvim_db", auto_execute = 1 },
   keys = {},
 
-  -- TODO why do we need this?
+  -- Give autocompletion for builtins that lack a config function
   builtin = {
-    lspinstall = {},
-    telescope = {},
-    compe = {},
-    autopairs = {},
-    treesitter = {},
-    nvimtree = {},
-    gitsigns = {},
-    which_key = {},
-    comment = {},
-    rooter = {},
-    galaxyline = {},
-    bufferline = {},
-    dap = {},
-    dashboard = {},
-    terminal = {},
+    lspinstall = {
```

Comment:
It would be very nice of you to provide the missing builtin modules, this way we could simplify this to `builtin={}`

Reply:
Done.